### PR TITLE
Various updates

### PR DESCRIPTION
**Pull request by drazik**
Some little things are grouped in this PR:

* Fix a typo in the nav in FR: « Securité » => « Sécurité »
* Fix the padding of the content: since we have an `overflow-y: scoll` on the content element, the padding-bottom was not rendered. Adding a intermediate element with the padding fixes it
* Update app name to "Pass" for both FR and EN
* Use a logo with the "beta" word
* Run tests in travis builds

### CODE REVIEW

File: src/components/App.jsx [hunk 1]
```diff
@@ -48,35 +48,37 @@ export const DumbApp = ({ breakpoints: { isDesktop } }) => {
...
-                  <Route
-                    path="/installation/installed"
-                    exact
-                    component={InstalledPage}
-                  />
-                  <Redirect from="/" to="/presentation" />
-                  <Redirect from="*" to="/presentation" />
-                </Switch>
+              <Content>
+                <div className="u-p-2 u-mt-auto u-mb-auto">
```
Comment (ptbrowne) [inline src/components/App.jsx]: Are we sure we do not want a margin-bottom here ? I have a bit of trouble understanding why the password app should have a custom layout.

Comment (drazik) [reply src/components/App.jsx]: The password app has its content vertically centered. That's why we have auto vertical margins.

Comment (ptbrowne) [reply src/components/App.jsx]: Are you sure that the password app will have all its pages with vertically centered content ? I think this is a coincidence that all the pages, as of now, are vertically centered. IMHO, the padding and vertical centering should not be hard coded in the general layout of the app, but preferably in a wrapper for all views.

Comment (drazik) [reply src/components/App.jsx]: I see your point, you're right. I'm doing this change right now.

Comment (drazik) [reply src/components/App.jsx]: See https://github.com/cozy/cozy-passwords/pull/15/commits/597ac9498d8cc098e4e7150618ef28fdcd8a074e

Comment (ptbrowne) [reply src/components/App.jsx]: Thanks !

**Review by ptbrowne — APPROVED** [review]

**Pull request merged**